pkg/minikube/extract/testdata/test.json

```diff
@@ -1,4 +1,7 @@
 {
+	"Hint: This is not a URL, come on.": "",
```

Comment:
Please revert this unrelated change. It's not your fault though - it's due to one of our new localization tests. Thanks!